src/SetZipper.elm

```diff
@@ -0,0 +1,19 @@
+module SetZipper exposing (Team, createZipper)
```

Comment:
This module name isn't very indicative.
Is this meant to be replaced by real TBA interaction?